api/core/v1/core.proto

```diff
@@ -162,14 +162,28 @@ message ObjectSelector {
     }
 }
 
-// TargetRef identifies Gateway API objects to apply a direct policy to.
+// PolicyTargetReference identifies Gateway API objects to directly apply policy to.
+// This is a copy of the upstream K8s Gateway API `targetRef` API.
+// See the following for more information:
+// * https://gateway-api.sigs.k8s.io/geps/gep-713/
+// * https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1alpha2.PolicyTargetReference
+// * https://github.com/kubernetes-sigs/gateway-api/blob/b4f0307cc9269e73187300e72979e7e111ab74ab/apis/v1alpha2/policy_types.go#L34-L56
+message PolicyTargetReference {
+  string group = 1;
+  string kind = 2;
+  string name = 3;
+
+  // Optional, if unspecified, the local namespace of the policy is inferred.
+  google.protobuf.StringValue namespace = 4;
+}
+
+// PolicyTargetReferenceWithSectionName identifies Gateway API objects, and optionally a specific section of those objects, to directly apply policy to.
 // This is a copy of the upstream K8s Gateway API `targetRef` API.
```

Comment:
this comment is out of date, should be PolicyTargetReferenceWithSectionName API?

Reply:
the intent was the call out the generic `targetRef` concept not the specific message it is using.
I can update this though